CMakeLists.txt

```diff
@@ -127,13 +127,13 @@ ELSE()
   IF (NOT KOKKOSKERNELS_HAS_TRILINOS AND NOT KOKKOSKERNELS_HAS_PARENT)
     # This is a standalone build
     FIND_PACKAGE(Kokkos REQUIRED)
-    IF((${Kokkos_VERSION} VERSION_EQUAL "4.1.00") OR (${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.2.00"))
+    IF((${Kokkos_VERSION} VERSION_EQUAL "4.2.00") OR (${Kokkos_VERSION} VERSION_GREATER_EQUAL "4.3.00"))
       MESSAGE(STATUS "Found Kokkos version ${Kokkos_VERSION} at ${Kokkos_DIR}")
-      IF((${Kokkos_VERSION} VERSION_GREATER "4.2.99"))
+      IF((${Kokkos_VERSION} VERSION_GREATER "4.3.99"))
         MESSAGE(WARNING "Configuring with Kokkos ${Kokkos_VERSION} which is newer than the expected develop branch - version check may need update")
       ENDIF()
     ELSE()
-      MESSAGE(FATAL_ERROR "Kokkos Kernels ${KokkosKernels_VERSION} requires 4.1.00, 4.2.00 or develop")
+      MESSAGE(FATAL_ERROR "Kokkos Kernels ${KokkosKernels_VERSION} requires 4.2.00, 4.3.00 or develop")
```

Comment:
To align with the comment above this would become
```
MESSAGE(FATAL_ERROR "Kokkos Kernels ${KokkosKernels_VERSION} requires Kokkos Core 4.2.00 or greater")
```

Reply:
👍